service/controller/cluster.go

```diff
@@ -69,10 +74,8 @@ func NewCluster(config ClusterConfig) (*Cluster, error) {
 
 			// Name is used to compute finalizer names. This here results in something
 			// like operatorkit.giantswarm.io/cluster-apps-operator-cluster-controller.
-			Name: project.Name() + "-cluster-controller",
-			Selector: labels.SelectorFromSet(map[string]string{
-				label.ClusterAppsOperatorVersion: project.Version(),
```

Comment:
My main question is: Can we use this version anywhere other than CAPO with this change?

Reply:
> What do you think about using a key-only label selector of label.ClusterAppsOperatorVersion?

@tfussell This should work provided there is only 1 instance of the operator from the collection. The key name is a bit strange if there is no value. Maybe it could be renamed?

Reply:
The new label is `label.ClusterAppsOperatorWatching` and it's now used here. Let me know what you think.